src/main/com/algos/src/maths/AbsoluteMax.java

```diff
@@ -13,12 +13,12 @@ public static int getMaxValue(int... numbers) {
         if (numbers == null || numbers.length == 0) {
             throw new IllegalArgumentException("Numbers array cannot be empty or null");
         }
-        int absMax = numbers[0];
+        int absMax = 0;  
         for (int i = 1; i < numbers.length; i++) {
-            if (Math.abs(numbers[i]) > Math.abs(absMax)) {
+            if (numbers[i] > absMax) {
                 absMax = numbers[i];
             }
         }
-        return absMax;
+        return Math.abs(absMax);
```

Comment:
**Performance Issue**: The change from using the absolute value comparison to a direct comparison when determining the maximum value can lead to incorrect results for negative numbers, impacting the algorithm's correctness and potentially its efficiency. <br> **Fix**: Restore the use of Math.abs for both the current number and the current maximum when comparing values inside the loop. This ensures that the algorithm correctly identifies the absolute maximum value, regardless of the sign of the input numbers. <br> **Code Suggestion**: 
 ```
 -        int absMax = 0;
 +        int absMax = numbers[0];
 -        for (int i = 1; i < numbers.length; i++) {
 -            if (numbers[i] > absMax) {
 +        for (int i = 0; i < numbers.length; i++) {
 +            if (Math.abs(numbers[i]) > Math.abs(absMax)) {
 -        return Math.abs(absMax);
 +        return absMax;
 ```